src/tests/unit/auto/log_utils_test.cpp

```diff
@@ -9,14 +9,17 @@
 #include <future>
 using ::testing::_;
 using namespace MockMultiDevice;
+// disable using windows.h
+#if 0
```

Comment:
is it for testing?
shall the code be finally removed?

Reply:
no, disable a test case about environment variable OPENVINO_LOG_LEVEL, the test case set environment variable at run time, so it need the header file "windows.h".  disable the test case temporarily

Reply:
Will remove "windows.h" and related debug logging in 2022.1 release for less risk on Windows platform. 

Will add debug logging and related test case from 2022.2.